src/UI/Shared/Grid/HeaderCell.js

```diff
@@ -17,13 +17,17 @@ module.exports = function() {
         render : function() {
             this.$el.empty();
             this.$el.append(this.column.get('label'));
+            if (this.column.get('tooltip')) {
+                this.$el.attr('title', this.column.get('tooltip'));
+                this.$el.attr('data-container', '.table');
```

Comment:
use an object to set both at the same time, less touching the DOM = more faster.